docs/core/tools/dotnet-publish.md

```diff
@@ -61,6 +61,10 @@ Specifies the path for the output directory. If not specified, it defaults to *.
 
 Configuration to use when building the project. The default value is `Debug`.
 
+`--manifest <PATH_TO_MANIFEST_FILE>`
```

Comment:
@mairaw I'll leave this for the next author or help from the team. I can't fix this in the time that remains.